### PR TITLE
Copy HGDP 65 samples

### PR DESCRIPTION
**Pull request by pdiakumis**
First attempt at a hail batch run via analysis-runner for copying those 65 HGDP CRAMs over.

### CODE REVIEW

**Comment by pdiakumis** [comment]
@lgruen this works when running in test (~12.5 MiB/s transfer speed):

```
analysis-runner \
  --access-level "test" \
  --dataset "fewgenomes" \
  --description "Test copy HGDP CRAM subset" \
  --output-dir "gs://cpg-fewgenomes-temporary/$(whoami)-copyCRAM-example" \
  copy_hgdp_crams.py
  ```
  
For writing into the `gs://cpg-fewgenomes-main` bucket, I'd need the `full` access level, right?

File: reports/hgdp_exploration/copy_hgdp_crams.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import hailtop.batch as hb
+
+# OUTPUT gets propagated from the analysis-runner cli to the server
+output_bucket = os.getenv('OUTPUT')
+assert output_bucket and output_bucket.startswith('gs://cpg-fewgenomes-temporary/')
+
+input_filelist = './data/filtered65.csv'
+analysis_runner_image = 'australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:45c3f8125e300cd70bb790e32d96816f003a7af2-hail-0.2.64.devcb1c44c7b529'
+
+def copy_to_bucket(bucket: str, batch: hb.batch.Batch, sample_name: str, ftype: str, fname: str) -> None:
```
Comment (lgruen) [inline reports/hgdp_exploration/copy_hgdp_crams.py:19]: This doesn't actually copy anything, so the name is a bit misleading. Maybe rename and add documentation on what this function does?

**Review by lgruen — CHANGES_REQUESTED** [review]

File: reports/hgdp_exploration/copy_hgdp_crams.py (hunk 1 above)
Comment (pdiakumis) [reply reports/hgdp_exploration/copy_hgdp_crams.py:19]: As in, it declares a Batch job that copies the given file into the given bucket?

File: reports/hgdp_exploration/copy_hgdp_crams.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+"""
+
+import os
+import csv
+import subprocess
+import hailtop.batch as hb
+
+# OUTPUT gets propagated from the analysis-runner cli to the server
+output_bucket = os.getenv('OUTPUT')
+assert output_bucket and output_bucket.startswith('gs://cpg-fewgenomes-temporary/')
```
[the remark this answers is not in the document]
Comment (pdiakumis) [reply reports/hgdp_exploration/copy_hgdp_crams.py:14]: D'oh, good catch

**Review by lgruen — APPROVED** [review]
Nice!

**Pull request merged**